index.html

```diff
@@ -14,15 +14,26 @@
     />
   </head>
   <body>
+    
     <header class="navbar">
       <a href="#" class="logo">My Logo</a>
-      <nav><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
+      <nav id = "menu-icon"><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
```

Comment:
- [ ] The menu icon is displayed perfectly and to be honest, it is doing its job 👏🏼 , but for semantic reasons we want the screen reader to inform the user that this is a clickable area 🖱️ so I suggest that the `img` tag should be wrapped inside `a` or `button` tags.

Reply:
done